packages/reports/tests/acceptance/schedule-modal-test.js

```diff
@@ -261,6 +282,10 @@ module('Acceptance | Navi Report Schedule Modal', function(hooks) {
       'Recipients field changes to a new schedule kept but not saved to the store'
     );
 
+    assert
+      .dom('.schedule-modal__must-have-data-toggle .x-toggle')
+      .isNotDisabled('notify when empty changes to an existing schedule are discarded after clicking cancel');
```

Comment:
Did you mean to check `isNotChecked` here instead of `isNotDisabled`?

Reply:
Good catch! Just changed it.